docs/Create.md

```diff
@@ -599,3 +601,90 @@ export default OrderEdit;
 ```
 
 **Tip:** If you'd like to avoid creating an intermediate component like `<CityInput>`, or are using an `<ArrayInput>`, you can use the [`<FormDataConsumer>`](./Inputs.md#linking-two-inputs) component as an alternative.
+
+## Controlled Mode
```

Comment:
Why *Controlled Mode*? This is misleading imo. Controlled vs Uncontrolled refers to other conceps

Reply:
I have no better name for it...

Reply:
With Default UI ?

Reply:
I don't understand what that means... I don't think it's better than "controlled mode"

Reply:
nvm